getting_started/intel_debug/introduction.rst

```diff
@@ -137,4 +137,36 @@ drivers and should work when changing the ``dsp_driver`` parameter.
 The selection of firmware, topology, and UCM files is based on platform
 capabilities, codec names, and DMI options. While the SOF team and the
 community try to cover all possible cases, errors will happen when the
-wrong file is selected at any of the three layers.
\ No newline at end of file
+wrong file is selected at any of the three layers.
+
+4. Chromebooks and SOF
+----------------------
+
+As stated above, Chromebook firmware is as of 2021 configured with the
+*community* key which means Chromebooks can run audio firmware signed by
+anyone. The entire filesystem is locked by default instead but there are
+a couple options to disable security for development purposes.
+
+Chromebooks can run Linux in several, non-mutually exclusive ways:
+
+- ChromeOS has direct hardware access but development cannot happen on
+  ChromeOS itself, it requires a separate workstation similar to how
+  most embedded development does. Setting up its ``cros_sdk`` requires
+  significant time and effort, see the `ChromeOS developer guide
+  <https://chromium.googlesource.com/chromiumos/docs/+/main/developer_guide.md>`_.
+
+- Crostini is a secure Linux Virtual Machine that does not have direct
+  access to the hardware and cannot be used for SOF.
+
+- Crouton is a non-secure chroot that does provide hardware access and
+  can be used for SOF. It lets you install a choice of popular Linux
+  distributions. The Zephyr project has `very detailed instructions
+  <https://docs.zephyrproject.org/2.7.0/boards/xtensa/intel_adsp_cavs25/doc/index.html>`_
+  on how to use Crouton for SOF. 
+
+- Finally, it is possible to "dual-boot" a regular Linux distribution on
+  *some* Chromebooks and forget it is a Chromebook entirely. This comes
+  at a price however: it is the least secure option and the more likely
+  to make your device permanently unusable ("brick"). If that does not
```

Comment:
how would it brick it? As long as the AP FW is intact there should not be as much risk.

Reply:
nvm it is a fw update i see